IS7024FinalProjectBreweryArc/BreweryArc/Pages/Index.cshtml.cs

```diff
@@ -29,23 +30,55 @@ public void OnGet()
                 using (var webClient = new WebClient())
                 {
                     // grab our JSON text.
-                    string jsonString = webClient.DownloadString("https://api.openbrewerydb.org/breweries");
+                    string Brewery_data = string.Empty;
+                    
+
+
+
+                    try
+                    {
+                        Brewery_data = webClient.DownloadString("https://api.openbrewerydb.org/breweries");
+                    }
+                    catch (Exception e)
+                    {
+                        Console.WriteLine("Exception while calling API", e);
+
+
+
+                    }
+                    
 
                     // convert raw text to objects.
-                    List<BreweryCollection> breweryCollections = BreweryCollection.FromJson(jsonString);
+                    List<BreweryCollection> breweryCollections = BreweryCollection.FromJson(Brewery_data);

```

Comment:
The name is more descriptive, but we typically don't use _ in variable names, unless it is a constant.  Name this breweryData or BreweryData.